packages/mdc-textfield/index.js

```diff
@@ -85,39 +85,39 @@ class MDCTextfield extends MDCComponent {
   }
 
   /**
-   * Initiliazes the Textfield's internal state based on the environment's
+   * Initiliazes the TextField's internal state based on the environment's
```

Comment:
"Text Field" in these comments as well.

While we're here, there's a typo (Initializes)